jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/overview/CaseOverviewViewImpl.html

```diff
@@ -41,7 +41,7 @@
                 <!-- actions -->
                 <div class="col-xs-12 col-sm-3 col-md-2 kie-case-view-header__actions">
       <span class="btn-group">
-          <button type="button" class="btn btn-primary" data-field="case-cancel" data-i18n-key="Cancel"></button>
+          <button type="button" class="btn btn-primary" data-field="case-destroy" data-i18n-key="Close"></button>
```

Comment:
Is there a particular reason why in the action behind the button and data-field attribute don't match i.e. _Close_ vs. _destroy_?

Reply:
@bsremac aligned a few more things based on this rename for consistency.